hack/import-bitbucket-server-projects.sh

```diff
@@ -0,0 +1,9 @@
+SERVER=$1
```

Comment:
@pgronkiewicz This can be used to pull all repos for projects in our BB. Then use this file as the input for the import command